### DISCUSSION TITLE
Questions: Do Helix has support to show type inference in Rust like VScode does?

### DISCUSSION THREAD

**Discussion by joaocarvalhoopen** [helix-editor/helix#4476] [Q&A] [answered]
Hello,

This is one of the most important features that VSCode has and I think VIM also has for Rust code development, because it let's you know all the time the types of all things in a easy and fast way.

Best regards,
João

**Comment by David-Else** [comment, marked as answer]
Not yet, in development: https://github.com/helix-editor/helix/pull/417

**Comment by Omnikar** [comment]
You can show the type and other information about the symbol under the cursor with `<space>k`.

**Reply by joaocarvalhoopen** [reply]
Thank you.
I'm currently doing the "hx --tutor",
And I will use helix to make it my dally driver for Rust.
But I hope that in a near future Helix can have the full  type anotation in place as VSCode has, because that feature is a really good feature.
My produtivity climed very much with that simple feature introduction in VSCode.
Thank you again,

Best regards,
João
